### DISCUSSION TITLE
Turtle of json-ld?

### DISCUSSION THREAD

**Discussion by NielsHoffmann** [bimloket/CB-NL#9] [General]
Michel stelt de vraag wat de serialisatie zou moeten zijn.
json-ld heeft als voordeel dat het meteen 'developer friendly' is...

**Comment by NielsHoffmann** [comment]
Ik zou zelf willen voorstellen om turtle als standaard te gebruiken. 
omzetten naar andere serialisaties is altijd simpel.

**Reply by AnoukVanOtterlo** [reply]
Ik denk dat turtle voor nu ook makkelijker is. Of hebben al veel mensen ervaring met json-ld? Dat zou eventueel kunnen helpen om daarvoor te kiezen.
